catalog/src/main/java/com/example/bnb/catalog/PostgresCatalogRepository.java

```diff
@@ -63,9 +63,10 @@ private void refreshCached() {
         Instant now = Instant.now();
```

Comment:
Edit this method to validate the freshness of the data we're caching, and error handling for cases where either the images or video_uri columns are missing. Make sure to include a custom StaleCacheDataException in the failure case in your commit suggestion